sklearn/utils/seq_dataset.pxd.tp

```diff
@@ -21,7 +21,7 @@ def get_dispatch(dtypes):
         yield name, c_type
 
 }}
-
+# cython: language_level=3
```

Comment:
I thought these need to be on top of the file.

Reply:
they are at the top of the cython file generated by this tp. but it wouldn't hurt to put it right at the top